test/askContainer/webview/profileManagerWebview.test.ts

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
+ *  Alexa Skills Toolkit for Visual Studio Code
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ *--------------------------------------------------------------------------------------------*/
+import * as assert from "assert";
+import * as sinon from "sinon";
+import { ProfileManagerWebview } from "../../../src/askContainer/webViews/profileManagerWebview";
+import {FakeExtensionContext, FakeWebviewPanelOnDidChangeViewStateEvent} from "../../testUtilities";
+import { EXTENSION_STATE_KEY } from "../../../src/constants";
```

Comment:
Some brackets having spacing and this one does not D: Which one is the default? 

Prettier should have caught this, not a big deal at all though :)

Reply:
small style nit: If possible I would organize it like so:

```
import * as assert from "assert";
import * as sinon from "sinon";
import * as vscode from "vscode";
import { ProfileManagerWebview } from "../../../src/askContainer/webViews/profileManagerWebview";
import { FakeExtensionContext, FakeWebviewPanelOnDidChangeViewStateEvent } from "../../testUtilities";
import { EXTENSION_STATE_KEY } from "../../../src/constants";
```

so we import other modules first then import our local code second

Reply:
Definitely agreed, that formatting looks cleaner. I've set up a pre-commit prettier hook for staged files using pretty-quick, so hopefully won't have much trouble keeping consistent formatting going forward.

Reply:
Fix commits are in.